### PR TITLE
support webpack.config.js "stats" from the CLI

### PR DESCRIPTION
**Pull request by pstoica**
I made "stats" the base object for outputOptions since there is no `Object.assign` available. Comment with any concerns. Thanks!

### CODE REVIEW

File: bin/webpack.js [hunk 1]
```diff
@@ -65,13 +65,11 @@ function ifArg(name, fn, init) {
...
 
+var outputOptions = options.stats || {};
 
-
-var outputOptions = {
-	cached: false,
-	cachedAssets: false,
-	context: options.context
-};
+outputOptions.cached = false;
```
Comment (sokra) [inline bin/webpack.js]: Only set them to `false` when not already specified.

Comment (pstoica) [inline bin/webpack.js]: Fixed. Is the CI failure a false negative or did something actually break?

**Comment by pstoica** [comment]
Hmm, I'm realizing that these sort of lines end up doing overrides:

```
if(!outputOptions.json) {
	ifArg("display-chunks", function(bool) {
		outputOptions.modules = !bool;
		outputOptions.chunks = bool;
	});
```

**Comment by sokra** [comment]
``` js
    // --display-chunks
    ifArg("display-chunks", function(bool) {
      if(bool) {
        outputOptions.modules = false;
        outputOptions.chunks = true;
      }
    });

    // the default
    if(typeof outputOptions.modules === "undefined")
      outputOptions.modules = true;

```

**Comment by pstoica** [comment]
Cheers!

**Pull request closed without merging**